packages/format-po/src/po.ts

```diff
@@ -60,7 +60,7 @@ function getCreateHeaders(language: string): PO["headers"] {
   }
 }
 
-const EXPLICIT_ID_FLAG = "explicit-id"
+const EXPLICIT_ID_FLAG = "js-lingui-explicit-id"
```

Comment:
Maybe the constant name should be changed as well?

```suggestion
const EXPLICIT_ID_COMMENT = "js-lingui-explicit-id"
```

Reply:
i suppose it still a flag but saved in comment?  for me naming is ok